shared/menubar/index.desktop.js

```diff
@@ -204,7 +207,7 @@ class MenubarRender extends React.Component<Props, State> {
             position="bottom right"
           />
         </Kb.Box>
-        <ChatContainer onViewAll={() => this.props.openApp(Tabs.chatTab)} />
+        <Chat.ChatContainer onViewAll={() => this.props.openApp(Tabs.chatTab)} conversations={this.props.conversations} onSelectConversation={this.props.onSelectConversation} />
```

Comment:
components in the widget can be connected so you don't really need to pass this stuff down, its just the store is a different shape than the main store